roles/rook/tasks/fetch-keys.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Include wait-for-cluster task
+  ansible.builtin.include_tasks: wait-for-cluster.yml
+  tags: wait
+
+
+- name: Fetch Ceph keyring for each client
+  kubernetes.core.k8s_info:
```

Comment:
Missing in osism/zuul-jobs, I will add this there.

Reply:
:+1: